chain/consensus/filcns/upgrades.go

```diff
@@ -2519,7 +2521,9 @@ func UpgradeActorsV14(ctx context.Context, sm *stmgr.StateManager, cache stmgr.M
 		return cid.Undef, xerrors.Errorf("migrating actors v14 state: %w", err)
 	}
 
-	log.Info("\n\n" + upgradeSplash)
+	if term.IsTerminal(int(os.Stderr.Fd())) {
+		fmt.Fprint(os.Stderr, upgradeSplash)
```

Comment:
FWIW, I would vote for removing the cosmetic splash in favour of consistently logging the system behaviour. 

In that, as a system administrator for a given version I want to be able to infer sequence of events as they happened whether I am looking at CLI output or logs. 

Assuming there is no concrete reason for the splash other than eye candy the most straight forward way I can think of to achieve this is to remove splash sugar, and simply log.

Having bored you fine people to tears, I have no strong opinions on this.